### ISSUE TITLE
libdcadec.dll adds offset to decoded data when used with frontend other than dcadec.exe

### ISSUE THREAD

**Issue by Rollinnn** [foo86/dcadec#51]
The problem is: on some dths-hd ma files decoding with libdcadec.dll controlled with other frontend than dcadec.exe(I used eac3to) gives resulted file with offsetted audio data.
You can see detailed description with samples and discussion in eac3to's thread on doom9, beginning from this my post: [http://forum.doom9.org/showthread.php?p=1758144#post1758144](http://forum.doom9.org/showthread.php?p=1758144#post1758144)

**Comment by Nevcairiel** [comment]
This is not a bug in libdcadec, the utility using this would need to parse the metadata and discard the decoded samples.

You have been told as much in the discussion you cite, as well.

**Comment by Rollinnn** [comment]
Yes, probably I look stupid, keeping complaining. But developer of foobar2000 plugin (foo_input_dts based on dcadec) said this is bug in dcadec. So I need official denial of bug in dcadec, so I'll can bother developer of foobar's plugin again.

**Comment by MarcusJohnson91** [comment]
You're all good @Rollinnn show him this issue page if he has questions.
